native-sql-engine/core/src/main/scala/com/intel/oap/expression/ColumnarBinaryExpression.scala

```diff
@@ -70,12 +70,37 @@ class ColumnarDateAddInterval(start: Expression, interval: Expression, original:
   }
 }
 
+class ColumnarUnixTimestamp(timeExp: Expression, format: Expression, original: UnixTimestamp)
+  extends UnixTimestamp(timeExp, format, original.timeZoneId)
+    with ColumnarExpression
+    with Logging {
```

Comment:
What types does this operator support? Could you add a buildCheck for supported types?

Reply:
Gazell already has ColumnarUnixTimestamp (convert to unix timestamp in sec).